drafts/use-cases/7.searching-events.md

```diff
@@ -0,0 +1,120 @@
+﻿## Searching for events
+
+### Story
+As an application user
+I want to be able to quickly search through my events
+So I find events I'm interested in.
+
+As an API consumer
+I want to be able to search through resources
+So I can find those interesting to me.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api/events?search=some text");
```

Comment:
As discussed on the call, let's elaborate this a bit. It would be great for a client to be able to discover the search affordance, enumerate and set the available parameters and then execute the request